cache/in-memory/src/event/member.rs

```diff
@@ -36,6 +36,7 @@ impl InMemoryCache {
         self.cache_user(Cow::Owned(member.user), Some(guild_id));
         let cached = CachedMember {
             avatar: member.avatar,
+            communication_disabled_until: None,
```

Comment:
Why `communication_disabled_until` is explicitly set to `None` instead of using `member.communication_disabled_until` ?